Misc/NEWS.d/next/Library/2019-11-04-17-20-43.bpo-38314.zWz6_P.rst

```diff
@@ -0,0 +1 @@
+The read pipe on UNIX transports now has access to the `is_reading()` API which is noted in the documentation for asyncio protocols as being generally available on read transports.
```

Comment:
I think we could make this more succinct and briefly mention that it's related to asyncio. 
```suggestion
The read pipe in :mod:`asyncio` UNIX transports now has access to the `is_reading()` API.
```